js-exercises/003_stringOperations.js

```diff
@@ -0,0 +1,48 @@
+// npm install readline-sync  --> for getting iput from terminal
+
+const readline = require("readline-sync");
+
+const reverseStr = str => {
+  return str
+    .split("")
+    .reverse()
+    .join("");
+};
+
+const mixLetters = (str1, str2) => {
+  let len = str1.length > str2.length ? str1.length : str2.length;
+  let combine = "";
+  for (let i = 0; len > i; i++) {
+    if (str1[i] == undefined) {
+      combine = combine + str2[i];
+    } else if (str2[i] == undefined) {
+      combine = combine + str1[i];
+    } else {
+      combine = combine + str1[i] + str2[i];
+    }
+  }
+  return combine;
+};
+
+const removeWhiteSpace = val => {
+  for (let i = 0; val.length > i; i++) {
+    if (val[i] == " ") {
+      val = val.replace(" ", "");
```

Comment:
why you need a for loop when you are using this function

Reply:
it is because of some string have multiple white-spaces.

    if (val[i] == " ") {
      val = val.replace(" ", "");
    }

above if function remove only first white spaces.

Reply:
write the program without replace function